Libraries/LibWeb/HTML/FormControlInfrastructure.cpp

```diff
@@ -121,10 +121,10 @@ WebIDL::ExceptionOr<Optional<Vector<XHR::FormDataEntry>>> construct_entry_list(J
             auto [x, y] = input_element->selected_coordinate();
 
             // 6. Create an entry with namex and x, and append it to entry list.
-            entry_list.append(XHR::FormDataEntry { .name = move(name_x), .value = String::number(x) });
+            entry_list.append(TRY(create_entry(realm, name_x, String::number(x), {})));
```

Comment:
You shouldn't need to pass the final argument to `create_entry()`, since it has a default value of `{}`.